modules/ingester/instance_search.go

```diff
@@ -609,23 +609,23 @@ func includeBlock(b *backend.BlockMeta, req *tempopb.SearchRequest) bool {
 // searchTagValuesV2CacheKey generates a cache key for the searchTagValuesV2 request
 // cache key is used as the filename to store the protobuf data on disk
 func searchTagValuesV2CacheKey(req *tempopb.SearchTagValuesRequest, limit int, prefix string) string {
-	query := req.Query
+	var cacheKey string
```

Comment:
you should probably leave these changes off. they will conflict when we attempt to merge this to main. only make the change in the live store?